cedar-lean/Cedar/Data/List.lean

```diff
@@ -56,9 +56,17 @@ theorem sizeOf_snd_lt_sizeOf_list {α : Type u} {β : Type v} [SizeOf α] [SizeO
   x ∈ xs → sizeOf x.snd < 1 + sizeOf xs
 := by
   intro h
-  have := List.sizeOf_lt_of_mem h
-  have : sizeOf x = 1 + sizeOf x.1 + sizeOf x.2 := rfl
-  omega
```

Comment:
Issue with `omega`: https://github.com/leanprover/lean4/issues/3848

Reply:
Interesting.  Do we know why this is a problem in the current version of Lean and wasn't in the older one we were using?  It sounds like this shouldn't have worked in the older version either but somehow did.

Reply:
On zulip, Joachim said: "There was a change where omega would no longer check atoms for definitional equality, merely syntactic equal."
But it sounds like there's also an issue with `omega`. I don't know enough to pretend to understand the root cause.